.github/workflows/ci-sharktank.yml

```diff
@@ -26,8 +26,17 @@ jobs:
     name: "Unit Tests and Type Checking"
     strategy:
       matrix:
-        version: [3.11]
-        os: [ubuntu-24.04, windows-2022]
+        python-version: ["3.11", "3.12"]
+        torch-version: ["2.3.0+cpu", "2.4.1+cpu", "2.5.1+cpu"]
```

Comment:
You could also omit the `+cpu` from the matrix and only add it back in
```
pip install ... torch==${{matrix.torch-version}}+cpu
```

4 more characters to use in the job names then :P

Reply:
Done.